torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2424,7 +2424,11 @@ def _init(
         # to: torch._subclasses.fake_tensor._ShapeEnvSettings
     ):
         if duck_shape is None:
-            duck_shape = config.use_duck_shape
+            # TODO: Delete and simplify this logic post-rollout
+            if not torch._utils_internal.justknobs_check("pytorch/dynamo:disable_default_duck_sizing"):
```

Comment:
This seems like a weird way to do it. In particular, if someone has explicitly set use_duck_shape, you are still going to override it with the change here. Instead, add None as a default value, and then switch the default (when nothing is set) between False and True based on the JK.